pkg/resolver/searchComplete.go

```diff
@@ -132,14 +135,63 @@ func (s *SearchCompleteResult) searchCompleteResults(ctx context.Context) ([]*st
 	}
 	defer rows.Close()
 	if rows != nil {
+		props := make(map[string]struct{})
 		for rows.Next() {
 			prop := ""
-			scanErr := rows.Scan(&prop)
+			var input interface{}
+			scanErr := rows.Scan(&input)
 			if scanErr != nil {
 				klog.Error("Error reading searchCompleteResults", scanErr)
 			}
-			srchCompleteOut = append(srchCompleteOut, &prop)
+			switch v := input.(type) {
+			case string:
+				// fmt.Println("prop type: string")
+				prop = v
+				props[v] = struct{}{}
+				// props = append(props, v)
+			case bool:
+				prop = strconv.FormatBool(v)
+				props[prop] = struct{}{}
+				// props = append(props, prop)
+			case float64:
+				prop = strconv.FormatInt(int64(v), 10)
+				props[prop] = struct{}{}
+
+			case map[string]interface{}:
+				// fmt.Println("prop type: array")
+
+				arrayProperties[s.property] = struct{}{}
+				// klog.Info("arrayProperties: ", arrayProperties)
+				// fmt.Println("prop type: map[string]interface")
+				// labelStrings := make([]string, 0, len(v))
+				for key, value := range v {
+					labelString := fmt.Sprintf("%v=%v", key, fmt.Sprintf("%v", value))
```

Comment:
This should be the same.
```suggestion
					labelString := fmt.Sprintf("%s=%s", key, value))
```